example/calculate_pi/pi_driver.cpp

```diff
@@ -88,6 +91,11 @@ parthenon::DriverStatus PiDriver::Execute() {
               << "PI = " << pi_val << "    rel error = " << (pi_val - M_PI) / M_PI
               << std::endl
               << std::endl;
+
+    std::fstream fs;
+    fs.open("summary.txt", std::fstream::out);
+    fs << "PI = " << pi_val << "    rel error = " << (pi_val - M_PI) / M_PI << std::endl;
```

Comment:
Nitpick: I don't love that the value of pi and the relative error are on the same line. I'd replace the 4 spaces with a newline. That's a little more human-readable IMHO.

Reply:
This has been addressed.